permissions-map/README.md

```diff
@@ -0,0 +1,213 @@
+HTTP/1.1 403 Forbidden
+Cache-Control: private
+Content-Type: application/json
+Couper-Error: access control error
+
+{
+  "error": {
+    "id":      "ca3s9qf56p5nbq9b5bog",
+    "message": "access control error",
+    "path":    "/calendars/",
+    "status":  403
+  }
+}
+```
+with the following log entry
+```
+access-control | {...,"error_type":"beta_insufficient_permissions","handler":"api","level":"error","message":"access control error: required permission \"calendar.readonly\" not granted","method":"GET",...
+```
+
+Or a successful request using the calendar scoped token:
+```
+$ curl -si -X GET localhost:8080/calendars/ -H "Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwic2NvcGUiOiJjYWxlbmRhciJ9.J6X4M5dk-ivyi9qpVu5HjGS9CC0J7e_7vReh6xLg_04aff5mE55Pk9B0WliT5cA4uDUybq9LgxgKEEKzOTQHFp3RW6TBDQQWxBCzaE0WyNj1nGO2ZNdvVIDk5QPHaj0-88bv-1fIdops3gzy0YFdA587kKDOnfjqCGT5tH07o2ZbzPp2ogLo-IkUNV_hYLp7zZvinJj3ve6Q3y-C08H1xJ_sPa3_141AO8lX-B3lTursZ4XXz-VxRDY9bNC0rizKEzPY0e2BJ2P8c1Q15EEdaWzSRzilPovLFGstTYmKjm4Wmf-6H9_Zlsj9Ax0wvu4sn2vlUhZK2qk-yPs4DllsUw"
+HTTP/1.1 200 OK
+Cache-Control: private
+Content-Type: application/json
+Granted-Permissions: calendar calendar.readonly calendar.events calendar.events.readonly
+Required-Permission: calendar.readonly
+Scope: calendar
+
+{"method":"GET","path":"/calendars/"}
+```
+
+If we try the `PATCH` method instead with the same token, we get:
+```
+$ curl -si -X PATCH localhost:8080/calendars/ -H "Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwic2NvcGUiOiJjYWxlbmRhciJ9.J6X4M5dk-ivyi9qpVu5HjGS9CC0J7e_7vReh6xLg_04aff5mE55Pk9B0WliT5cA4uDUybq9LgxgKEEKzOTQHFp3RW6TBDQQWxBCzaE0WyNj1nGO2ZNdvVIDk5QPHaj0-88bv-1fIdops3gzy0YFdA587kKDOnfjqCGT5tH07o2ZbzPp2ogLo-IkUNV_hYLp7zZvinJj3ve6Q3y-C08H1xJ_sPa3_141AO8lX-B3lTursZ4XXz-VxRDY9bNC0rizKEzPY0e2BJ2P8c1Q15EEdaWzSRzilPovLFGstTYmKjm4Wmf-6H9_Zlsj9Ax0wvu4sn2vlUhZK2qk-yPs4DllsUw"
+HTTP/1.1 405 Method Not Allowed
+Cache-Control: private
+Content-Type: application/json
+Couper-Error: method not allowed error
+
+{
+  "error": {
+    "id":      "ca3sbfn56p5nbq9b5bpg",
+    "message": "method not allowed error",
+    "path":    "/calendars/",
+    "status":  405
+  }
+}
+```
+and the following log entry:
+```
+access-control | {...,"handler":"api","level":"error","message":"method not allowed error: method PATCH not allowed by beta_required_permission","method":"PATCH",...
+```
+This happens because the `PATCH` method is not mentioned in the `beta_required_permission` attribute value (neither explicitly, nor implicitly via "*").
```

Comment:
```suggestion
This happens because the `PATCH` method is not mentioned in the `beta_required_permission` attribute value (neither explicitly, nor implicitly via `"*"`).
```